content/sensu-core/0.29/installation/configuration-managment.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Configuration Management"
+description: "Configuration Management with Sensu"
+weight: 16
+product: "Sensu Core"
+version: "0.29"
+menu:
+  sensu-core-0.29:
+    parent: installation
+---
+
+We highly recommend using configuration management tools to deploy Sensu in production and at scale.
+
+* Pin versions of Sensu-related software to avoid issues due to version drift as new Sensu components are deployed.
```

Comment:
I'm a bit confused here. Who is deploying new Sensu components and why is version drift happening?

Reply:
So the who would be some company using Sensu. And the version drift can occur as you build new servers and then install Sensu, but you don't specify a version when you do so, which then leads to the latest getting installed instead of the version you have on say your already running servers.